kinto/plugins/kinto_admin/tests/__init__.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+import webtest
+from kinto.core import utils as core_utils
+try:
+    from kinto.core import testing as core_support
+except ImportError:
```

Comment:
I guess we can remove that since we are now in the Kinto directory we will support the current version of Kinto.